samples/lj-demo.py

```diff
@@ -182,6 +183,9 @@ class Controls(HasTraits):
     default_output = default_output if len(
         outputs) > 1 else through_port_output
 
+    if default_input is None or default_output is None:
+        raise RuntimeError('Cannot connect to any MIDI device')
```

Comment:
This shouldn't raise an exception as someone might want to run the demo without a MIDI controller. Instead, just print a message.

Reply:
done